### PR TITLE
Minor improvements (double default, no awk)

### PR DESCRIPTION
**Pull request by cmeury**
I was annoyed by "default" showing up twice, so this PR should fix that. Let me know what you think.

### CODE REVIEW

File: kns [hunk 1]
```diff
@@ -13,12 +13,13 @@ if [ ! -x "$(which fzf 2>/dev/null)" ]; then
...
 
-current="$(kubectl config current-context)"
-namespace="$(kubectl config get-contexts "$current" | awk "/$current/ {print \$5}")"
+export current="$(kubectl config current-context)"
+namespace="$(kubectl config view -o jsonpath="{.contexts[?(@.name == '${current}')].context.namespace}")"
 if [ -z "$namespace" ]; then
   namespace="default"
 fi
-selected=$( (kubectl get namespaces -o name | sed "s-namespaces/--"; echo $namespace ) | fzf -0 -1 --tac -q "${1:-""}" --prompt "$current> " --preview "kubectl --namespace {} get pods")
+export namespace
```
Comment (cmeury) [inline kns]: I might have to safe the original namespace environment variable here to not cause the script to mess up the calling environment. But it did not find out how to properly do the jsonpath extraction without a shell variable.

**Comment by iMartyn** [comment]
@koenbollen This is a good addition and fixes 1.10 compatibility, any chance it could be merged?

**Comment by koenbollen** [comment]
Hey! Thanks for you contribution! Sorry for the late reaction, I was off for a week.

Change look good! I'll get it merged and update the homebrew formula in our tap. 👌 

Cheers!

**Comment by cmeury** [comment]
Thanks a lot @koenbollen! Appreciate the merge.

**Pull request merged**